src/view/com/post-thread/PostThread.tsx

```diff
@@ -449,39 +448,49 @@ export function PostThread({
   }
 
   return (
-    <ScrollProvider onMomentumEnd={onMomentumEnd}>
-      <List
-        ref={ref}
-        data={posts}
-        renderItem={renderItem}
-        keyExtractor={keyExtractor}
-        onContentSizeChange={isNative ? undefined : onContentSizeChangeWeb}
-        onStartReached={onStartReached}
-        onEndReached={onEndReached}
-        onEndReachedThreshold={2}
-        onScrollToTop={onScrollToTop}
-        maintainVisibleContentPosition={
-          isNative ? MAINTAIN_VISIBLE_CONTENT_POSITION : undefined
-        }
-        // @ts-ignore our .web version only -prf
-        desktopFixedHeight
-        removeClippedSubviews={isAndroid ? false : undefined}
-        ListFooterComponent={
-          <ListFooter
-            // Using `isFetching` over `isFetchingNextPage` is done on purpose here so we get the loader on
-            // initial render
-            isFetchingNextPage={isFetching}
-            error={cleanError(threadError)}
-            onRetry={refetch}
-            // 300 is based on the minimum height of a post. This is enough extra height for the `maintainVisPos` to
-            // work without causing weird jumps on web or glitches on native
-            height={windowHeight - 200}
-          />
-        }
-        initialNumToRender={initialNumToRender}
-        windowSize={11}
-      />
-    </ScrollProvider>
+    <CenteredView style={[a.flex_1]} sideBorders={true}>
```

Comment:
We need to wrap this in a centered view and use _those_ side borders, rather than the ones on the `List`. Unless we tweaked `List` again to add a `header` prop, which just seems a little overkill since this is the only screen we need it on.